### PR TITLE
Validator is off when MODE is not PROD

### PR DESCRIPTION
**Pull request by lruzicki**
Password Validator is off when MODE is not PROD

### CODE REVIEW

**Review by delcroip — CHANGES_REQUESTED** [review]
please transform that into a valid AUTH_PASSWORD_VALIDATORS

**Comment by lruzicki** [comment]
> please transform that into a valid AUTH_PASSWORD_VALIDATORS

@delcroip thanks for advice, I did change it. Take a look

File: core/utils.py [hunk 1]
```diff
@@ -370,32 +371,45 @@ def validate_json_schema(schema):
...
+
+    def validate(self, password, user=None):
+        if os.environ.get("MODE", "DEV") != "PROD":
+            if not self.schema.validate(password):
+                raise ValidationError(
+                    f"Password must be at least {settings.PASSWORD_MIN_LENGTH} characters long, "
+                    f"have at least {settings.PASSWORD_UPPERCASE} uppercase letter(s), "
+                    f"{settings.PASSWORD_LOWERCASE} lowercase letter(s), "
+                    f"{settings.PASSWORD_DIGITS} number(s), and "
+                    f"{settings.PASSWORD_SYMBOLS} special character(s)."
```
Comment (delcroip) [inline core/utils.py:399]: that should avoid code duplications

```suggestion
                    self.get_help_text()
```

File: setup.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
         'django-dirtyfields',
         'websocket-client',
         'jsonschema',
+        'password-validator'
     ],
```
Comment (delcroip) [inline setup.py:33]:
```suggestion
        'password-validator',
        'zxcvbn'
    ],
```

**Review by delcroip — APPROVED** [review]

**Pull request merged**